udata_gouvfr/templates/faq/developer.html

```diff
@@ -46,18 +46,32 @@ <h2 id="embed">Intégrer des cartes sur vos propres sites</h2>
 <pre>
 <code>
 &lt;div data-udata-dataset-id="ID DU JEU DE DONNÉES"&gt;&lt;/div&gt;
-&lt;script src="https://www.data.gouv.fr/static/widgets.js" id="udata" async defer&gt;&lt;/script&gt;
+&lt;script src="https://www.data.gouv.fr/static/widgets.js" id="udata" async defer onload="udataScript.load()"&gt;&lt;/script&gt;
 </code>
 </pre>
 
+<p>Vous pouvez récupérer l’ID DU JEU DE DONNÉES en cliquant sur le bouton des <em>Détails</em> lorsque vous êtes sur la page d’un jeu de données, il sera de la forme : <code>549d48f9c751df513b04805b</code>.</p>
```

Comment:
A markup to emphase maybe better that upper case (ie. `<em>`, `<strong>` or `<b>`)